ydb/core/tx/schemeshard/schemeshard_info_types.cpp

```diff
@@ -50,6 +51,16 @@ EDiskUsageStatus CheckStoragePoolsQuotas(const THashMap<TString, TStoragePoolUsa
             : EDiskUsageStatus::BelowSoftQuota;
 }
 
+EUserFacingStorageType GetUserFacingStorageType(const TString& poolKind) {
+    if (poolKind.StartsWith("ssd")) {
```

Comment:
This should cover "ssd" and "ssdencrypted". This hack will certainly be documented.

Reply:
Also there should be LOUD (but short if possible) indication and explanation that this is a workaround.
That makes sense only in a limited case when there are no multiple storage-pools with the same kind and storage-pool kinds are all in (ssh*, hdd*, rot*, nvme*) set.